backend/app/api/websocket.py

```
@@ -0,0 +1,53 @@
+import redis.asyncio as redis
+from app.config import settings
+import json
+import asyncio
+
+router = APIRouter(tags=["WebSocket"])
+
+
+@router.websocket("/ws/notifications")
+async def websocket_notifications(
+    websocket: WebSocket,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)